index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Is this html file necessary? It doesn't seem to serve any purpose. You can use node.js to run your script.

```bash
node script.js

// output logged to terminal
```

Reply:
I was initially working with a different approach that had a design, but I settled with the current method and I just kept the HTML file.